projects/rero/ng-core/src/lib/record/search/aggregation/pipe/bucket-name.pipe.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2022 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'bucketName'
+})
+export class BucketNamePipe implements PipeTransform {
+
+  /**
+   * Tranform value if the type of aggregation is language
+   * @param value - aggregation value
+   * @param aggregationKey - aggregation type
+   * @returns string
+   */
+  transform(value: string, aggregationKey: string): string {
+    if ('language' !== aggregationKey) {
+      return value;
+    }
+    return `lang_${value}`;
```

Comment:
I will suggest a 'switch' here. At this time, only 'language' aggregation are special process but how knows in future... Additionally, it will be more readable to use `equals` comparaison instead of `difference`.
Your solution should works well, but my suggestion is : 
```typescript
switch(aggregationKey) {
    case 'language': return `lang_${value}`;
    //case 'foo': return 'bar';
    default: return value;
}
```